### DISCUSSION TITLE
Error: Hessian could not be computed

### DISCUSSION THREAD

**Discussion by rifatzahan** [cdriveraus/ctsem#49] [General]
I tried running both the chicken diet model, as well as, a well-being and smartphone use model, that you shared the code online. However, while working on this, 
`#fit the model
fit_frequency<-ctStanFit(datalong = data_freq, ctstanmodel = model_tot, plot=10,verbose=0,cores=cores,nopriors=F)`, 
the model starts running and then throws the following error:

`Maximum a posteriori estimation requested
Individual variation in T0VAR parameters not possible, removed
Using 6/8 logical CPU cores
starting worker pid=68659 on localhost:11022 at 00:30:00.032
starting worker pid=68681 on localhost:11022 at 00:30:00.285
starting worker pid=68711 on localhost:11022 at 00:30:00.533
starting worker pid=68738 on localhost:11022 at 00:30:00.763
starting worker pid=68761 on localhost:11022 at 00:30:01.000
starting worker pid=68779 on localhost:11022 at 00:30:01.237
Doing 1st pass optimize...
.................................................................................................
Error in .local(object, ...) : 
  the model object is not created or not valid
Error in .local(object, ...) : 
  the model object is not created or not valid
Warning in FUN(X[[i]], ...) : NA gradient encountered at param 104
Problems computing Hessian...
Error in eigen(if (doDykstra) R else Y, symmetric = TRUE) : 
  infinite or missing values in 'x'
Error in (function (standata, sm, init = "random", initsd = 0.01, sampleinit = NA,  : 
  Hessian could not be computed
Optimization cancelled -- restart from current point by including this argument:
init = c(0.003, 0.00522, 0.01099, 0.00609, 0.0045, -0.00096, -0.00962, 0.00591, 0.00441.....................)`

Any idea as to what's going on? Or how I can solve this issue? Thanks in advance.

**Comment by cdriveraus** [comment]
You seem to have some general problem with the installation, most likely
related to rstan / stanheaders. I would try removing those as well as
ctsem, reinstall (not from source if it asks you) and see how things go.

**Reply by rifatzahan** [reply]
Yes, it worked. Thanks.
